components/symbol/examples/40-local-tokens.js

```diff
@@ -0,0 +1,55 @@
+/** @jsx jsx */
+
+import { GEL, jsx } from '@westpac/core';
+import {
+	MastercardAcceptedSymbol,
+	BPayLandSymbol,
+	AppleStoreSymbol,
+	WBCLogo,
+	STGLogo,
+} from '@westpac/symbol';
+import { Cell, Grid, Name } from './_utils';
+
+const Wrapper = ({ children, ...props }) => (
+	<div
+		css={{
+			border: '2px solid red',
+			padding: '24px',
+			height: '100%',
+		}}
+	>
+		{children}
```

Comment:
```suggestion
		{children} - {symbol}
```

Reply:
Let's show in the examples that you have access to this `symbol` prop now in the `Wrapper`.